test/org/javarosa/core/model/utils/test/DateUtilsTests.java

```diff
@@ -331,6 +334,40 @@ private void testCycle(Date in) {
 		}
 
 	}
+
+	public void testFormatting() {
+	    class LangJanSun {
+            private LangJanSun(String language, String january, String sunday) {
+                this.language = language;
+                this.january = january;
+                this.sunday = sunday;
+            }
+
+            private String language;
+	        private String january;
+	        private String sunday;
+        }
+
+        LangJanSun langJanSuns[] = new LangJanSun[] {
```

Comment:
All the possible values of the class are predefined and constant so I'd say it follows the letter of the docs.  Either way, apologies if I caused offence.